reactnative-todo/.env.example.js

```diff
@@ -0,0 +1 @@
+const API_URL = ''
```

Comment:
Could we just move all of the variables from `.env`, even those related to the Node.js server needs?